util/global_community_silence.php

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env php
+<?php
+
+/**
+ * @brief tool to silence accounts on the global community page
+ *
+ * With this tool, you can silence an account on the global community page.
+ * Postings from silenced accounts will not be displayes on the community
+ * page. This silencing does only affect the display on the community page,
+ * accounts following the silenced accounts will still get their postings.
+ *
+ * Usage: pass the URL of the to be silenced account as only parameter
+ *        at the command line when running this tool. E.g.
+ *
+ *        $> util/global_community_silence.php http://example.com/profile/bob
+ *
+ *        will silence bob@example.com so that his postings wont appear at
+ *        the global community page.
+ *
+ * Author: Tobias Diekershoff
+ *
+ * License: AGPLv3 or later, same as Friendica
+ **/
+
+if ($argc!=2 || $argv[1]=="-h" || $argv[1]=="--help" || $argv[1]=="-?") {
+	echo "Usage: ".$argv[0]." [-h|profile_url]\r\n";
+	echo "    -h, -?, --help ... show this help\r\n";
+	echo "    profile_url ...... The URL of the profile you want to silence\r\n";
+	echo "\r\n";
+	echo "Example: Silence bob@example.com\r\n";
+	echo "$> ".$argv[0]." https://example.com/profiles/bob\r\n";
+	echo "\r\n";
+	exit(0);
+}
+
+use Friendica\Database\DBM;
+use Friendica\Network\Probe;
+
+require_once("boot.php");
+require_once('include/dba.php');
+require_once("include/text.php");
```

Comment:
Standards: Please decide between single or double-quotes for the whole file. Single quotes are preferred.